ds3-autogen-parser/src/main/resources/typeNameMap.json

```diff
@@ -74,7 +74,7 @@
     },
     {
       "ContractName": "NodeApiBean",
-      "SdkName": "Ds3Node"
+      "SdkName": "Endpoint"
```

Comment:
I'm not sure if renaming the Ds3Node -> Endpoint is clear. Maybe rename to NodeDetails instead...?